### ISSUE TITLE
Tasks are not restarting nor scaling

### ISSUE THREAD

**Issue by rupamroy** [mesosphere/marathon#3791]
Hi 
I am very new to Mesos/Marathon - I am following the following set of tutorials
https://open.mesosphere.com/advanced-course/using-the-marathon-gui/

When i start a task (command `python -m SimpleHTTPServer $PORT`) , it starts up correctly , but if i restart this task or try to scale up from 1 ==> 2 , It hangs up waiting on deployment. There is no logs and no way to tell what is going on.  

Here is a screen shot of where is is stuck:
![image](https://cloud.githubusercontent.com/assets/9621769/14589219/6bb810d0-04aa-11e6-822f-1c8d47198ca0.png)

**Comment by vishnu2kmohan** [comment]
We'll need to see your app's complete JSON definition and it would help to share how may agents and the resources they each have (and the roles to which they belong).

**Comment by rupamroy** [comment]
Here is the complete json definition for my application

{
  "id": "/http-simple",
  "cmd": "python -m SimpleHTTPServer $PORT",
  "cpus": 1,
  "mem": 32,
  "disk": 0,
  "instances": 1,
  "healthChecks": [
    {
      "path": "/",
      "protocol": "HTTP",
      "portIndex": 0,
      "gracePeriodSeconds": 300,
      "intervalSeconds": 60,
      "timeoutSeconds": 20,
      "maxConsecutiveFailures": 3,
      "ignoreHttp1xx": false
    }
  ],
  "portDefinitions": [
    {
      "port": 10000,
      "protocol": "tcp",
      "labels": {}
    }
  ]
}

**Comment by aquamatthias** [comment]
@rupamroy: See here what Waiting means: https://mesosphere.github.io/marathon/docs/waiting.html
You probably run out of resources?

**Comment by vishnu2kmohan** [comment]
@rupamroy Are you using DC/OS from dcos.io ?

Where is python installed on your agents? When you a {{which python}} does it show {{/opt/mesosphere/bin/python}}? If so, please try specifying the full path to the python interpreter and change {{SimpleHTTPServer}} to {{http.server}} since we install and use python3 by default.

**Comment by njappboy** [comment]
@rupamroy I'm assuming you got past this issue. I ran into as well and @aquamatthias hit the nail on the head for me.  I was previously working with DC/OS v1.7 which OOTB set Marathon application defaults to CPU usage of 0.1. When working through the [https://open.mesosphere.com/advanced-course/using-the-marathon-gui/](https://open.mesosphere.com/advanced-course/using-the-marathon-gui/) tutorial I failed to immediately notice that Marathon application defaults to 1 CPU for the application. Once I changed it to 0.1 for the simple python server I was able to scale out as per the tut.
